modules/test-runner/src/transfer/asyncTransferOffline.test.ts

```diff
@@ -94,7 +106,8 @@ describe("Async transfer offline tests", () => {
    * Avoid `transfer.set-recipient.${this.userPublicIdentifier}` endpoint
    * with the hub. The hub should clean these up during disputes
    */
-  it("sender successfully installs transfer, goes offline before sending paymentId or preimage, then comes online and has the pending installed transfer", async () => {
+  // TODO: this doesn't work bc `transfer.set-recipient` isn't used anymore
+  it.skip("sender successfully installs transfer, goes offline before sending paymentId or preimage, then comes online and has the pending installed transfer", async () => {
```

Comment:
i dont think that we need this test anymore -- as soon as the sender has the app installed, it should be okay. theres no way for this failure to occur, and the positive case has been tested when the sender is offline and the receiver comes back online

Reply:
Perfect I love deleting stuff lol.